src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -22,26 +24,42 @@ public static String buildConceptChildrenQuery(
     FieldVariable nameFieldVariable = conceptTableVariable.makeFieldVariable("concept_name");
     FieldVariable idFieldVariable = conceptTableVariable.makeFieldVariable("concept_id");
 
-    TablePointer ancestorTablePointer =
-        TablePointer.fromTableName("concept_ancestor", tableNameGenerator);
-    TableVariable ancestorTableVariable = TableVariable.forPrimary(ancestorTablePointer);
+    TablePointer relationshipTablePointer =
+        TablePointer.fromTableName("concept_relationship", tableNameGenerator);
+    TableVariable relationshipTableVariable = TableVariable.forPrimary(relationshipTablePointer);
     FieldVariable descendantFieldVariable =
-        ancestorTableVariable.makeFieldVariable("descendant_concept_id");
+        relationshipTableVariable.makeFieldVariable("concept_id_2");
 
-    BinaryFilterVariable whereClause =
-        new BinaryFilterVariable(
-            ancestorTableVariable.makeFieldVariable("ancestor_concept_id"),
-            BinaryFilterVariable.BinaryOperator.EQUALS,
-            new Literal(conceptId));
+    BooleanAndOrFilterVariable whereClause =
+        new BooleanAndOrFilterVariable(
+            BooleanAndOrFilterVariable.LogicalOperator.AND,
+            List.of(
+                new BinaryFilterVariable(
+                    relationshipTableVariable.makeFieldVariable("concept_id_1"),
+                    BinaryFilterVariable.BinaryOperator.EQUALS,
+                    new Literal(conceptId)),
+                new BinaryFilterVariable(
+                    relationshipTableVariable.makeFieldVariable("relationship_id"),
+                    BinaryFilterVariable.BinaryOperator.EQUALS,
+                    new Literal("Subsumes"))));
     Query subQuery =
-        new Query(List.of(descendantFieldVariable), List.of(ancestorTableVariable), whereClause);
-    SubQueryFilterVariable subQueryFilterVariable =
-        new SubQueryFilterVariable(idFieldVariable, SubQueryFilterVariable.Operator.IN, subQuery);
+        new Query(
+            List.of(descendantFieldVariable), List.of(relationshipTableVariable), whereClause);
+    FilterVariable filterVariable =
+        new BooleanAndOrFilterVariable(
+            BooleanAndOrFilterVariable.LogicalOperator.AND,
+            List.of(
+                new SubQueryFilterVariable(
+                    idFieldVariable, SubQueryFilterVariable.Operator.IN, subQuery),
+                new BinaryFilterVariable(
+                    conceptTableVariable.makeFieldVariable("standard_concept"),
+                    BinaryFilterVariable.BinaryOperator.EQUALS,
+                    new Literal("S"))));
     Query query =
         new Query(
             List.of(nameFieldVariable, idFieldVariable),
             List.of(conceptTableVariable),
-            subQueryFilterVariable);
+            filterVariable);
     return query.renderSQL(platform);
```

Comment:
It may be helpful for readers to have the SQL commented before `buildConceptChildrenQuery`

Example (from the test you wrote): 
```
        SELECT c.concept_name, c.concept_id
        FROM concept AS c
        WHERE (c.concept_id IN
          (SELECT c.concept_id_2 FROM concept_relationship AS c
          WHERE (c.concept_id_1 = ... AND c.relationship_id = 'Subsumes')) AND c.standard_concept = 'S')"""
          ```